robyn/types.py

```diff
@@ -18,5 +18,32 @@ def as_list(self):
         ]
 
 
+PathParams = NewType("PathParams", Dict[str, str])
+Method = NewType("Method", str)
+FormData = NewType("FormData", Dict[str, str])
+Files = NewType("Files", Dict[str, bytes])
+IPAddress = NewType("IPAddress", Optional[str])
+
+
 class JSONResponse(TypedDict):
+    """
+    A type alias for openapi response bodies
+    """
+
+    pass
+
+
+class Body:
+    """
+    A type alias for openapi request bodies
+    """
+
+    pass
+
+
+class QueryParams:
+    """
+    A type alias for query params type annotations
+    """
+
```

Comment:
also you should implement the method definitions here. Associated with rust query params.